### ISSUE TITLE
Accessibility - Individual Goods Screen - Application Does Not Remove All Content when Changing Exploitation Types

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#48]
When going to “Beans (green, soy, yellow)’ individual goods screen and changing the ‘Exploitation Type’ to ‘Child Labor’ it still shows the country of ‘Burma’ even though it is not one of the types of exploited labor for it. Not sure if this good should have all three types of exploited labor but only two of them are shown on the screen. If you go to the ‘Beef’ screen and chose ‘Forced Labor’ then the one and only country of ‘Brazil’ goes away as expected.

**Comment by aquilent-dgemoets** [comment]
This is working as intended and is the same behavior as on iOS.  Forced Child Labor is considered a subset of Child Labor.

**Comment by aquilent-dgemoets** [comment]
Should this be closed, or is it still considered an issue?  If so, it is an issue in iOS as well.
